eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/PostRestartHtlcCleaner.scala

```diff
@@ -251,6 +251,8 @@ class PostRestartHtlcCleaner(nodeParams: NodeParams, register: ActorRef, initial
 
 object PostRestartHtlcCleaner {
 
+  type ChannelIdAndHtlcId = (ByteVector32, Long)
```

Comment:
I've decided to add this because I've had troubles immediately understanding what exactly `(ByteVector32, Long)` represents.

Reply:
But you're only doing half of the job, because the function declaration in `PluginParams` returns a `Set[(ByteVector32, Long)]`.
This needs to be consistent, either you introduce a `case class HtlcAndChannelIds` and use it everywhere, or you keep `(ByteVector32, Long)` (which IMHO is good enough).